app/api/schema/sessions.py

```diff
@@ -83,6 +88,7 @@ def validate_fields(self, data, original_data):
     ends_at = fields.DateTime(allow_none=True)
     language = fields.Str(allow_none=True)
     slides_url = fields.Url(allow_none=True)
+    multiple_slides_url = fields.Nested(DocumentLinkSchema, many=True, allow_none=True)
```

Comment:
Rename this to `slides`. Change application logic wherever required to now use this field instead of `slides_url`. And remove `slides_url` from models and update the migration in this PR itself.